### DISCUSSION TITLE
Third Party Hardware Compatible with Axis Camera Station

### DISCUSSION THREAD

**Discussion by djpjd** [AxisCommunications/discussions#233] [General]
Does anyone know of a list of third party hardware that works with ACS?  I am looking for a WIFI Camera to do some temporary monitoring and Axis has no option for me in their hardware offerings.  I have tried a few ONVIF devices but they do not pass the test on the Device Compatibility Tool.  I would prefer to purchase a sure thing as opposed to keep trying, failing and returning others gear.

**Comment by vivekatoffice** [comment]
Related link:
[AXIS Camera Station Device Compatibility Tool](https://www.axis.com/products/axis-camera-station-device-compatibility-tool)
